src/components/Message.tsx

```diff
@@ -1,16 +1,39 @@
 import React from "react";
 import { isLinkValid } from "../functions/isLinkValid";
+import { isLinkDuplicate } from "../functions/isLinkDuplicate";
 import { UrlType } from "../types/Url";
+import { ListItem } from "../types/ListItem";
 
-type MessageProps = UrlType & {};
+type MessageProps = UrlType & {
+  items: ListItem[];
+};
 
 export class Message extends React.Component<MessageProps> {
   render() {
-    const isValid = isLinkValid(this.props.url);
-    if (isValid) {
-      return <p>v pho</p>;
+    if (this.props.url) {
+      if (/(http(s?)):\/\//i.test(this.props.url)) {
```

Comment:
Would be nice to extract this to the method for better readability similar to `isLinkValid`